### PR TITLE
London class 8 - Shadi Safaee - HTML CSS Coursework - Week2

### PR DESCRIPTION
**Pull request by ShadiSafaee**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: css/store.css [hunk 1]
```diff
@@ -0,0 +1,285 @@
+* {
```
Comment (ellietms) [inline css/store.css:1]: Hi @ShadiSafaee, I can see you created 2 pull requests and both of them are for week2 homework and the other PR has been reviewed, is this PR extra?
please let me know. so I can close or review this PR :) 
Thanks

Comment (ShadiSafaee) [reply css/store.css:1]: Hi @ellietms,

This is extra, created by mistake 
https://github.com/ShadiSafaee/HTML-CSS-Coursework-Week2/pulls

Comment (ellietms) [reply css/store.css:1]: Hi @ShadiSafaee, This current PR is extra? (Is this your week2 home work?) 
https://github.com/CodeYourFuture/HTML-CSS-Coursework-Week2/pull/336

Comment (ShadiSafaee) [reply css/store.css:1]: @ellietms I think this has happened as I created a new branch for this week homework, whatever comes under "store" name is related to this week. So, I`m not sure which one can be deleted as my TA has commented the earlier version and it`s good for me to keep them. Any idea?

**Review by beckyf0keefe — COMMENTED** [review]
Overall looking good, really good understanding of the form stuff, just make sure you are really having an eye for detail with the design. Overall the CSS is good, great that you took onboard my comment about not using #ids, just would be great to see a little more focus on making it really match the design, a little padding and margin go along way!

**Pull request closed without merging**